common/src/main/java/com/google/tsunami/common/command/CommandExecutor.java

```diff
@@ -100,6 +105,19 @@ public String getError() {
     return error;
   }
 
+  public void setInput(String input) {
```

Comment:
Hm, it'd be nicer if we could set this in the constructor along with the arguments, though we can't easily do that atm because the constructor accepts "String...".
It's probably fine to leave this as is for now, but in the future we might want to change the constructor to something like CommandExecutor(String[] args, String input="") (and update the plugins to use the new constructor).